src/services/ledger/index.types.d.ts

```diff
@@ -76,20 +70,26 @@ type SendAmountsMetadata = {
   displayCurrency: DisplayCurrency
 }
 
-type AddLnSendLedgerMetadata = NonIntraledgerLedgerMetadata &
+type AddLnSendLedgerMetadata = LedgerMetadata &
   SendAmountsMetadata & {
     hash: PaymentHash
     pubkey: Pubkey
     feeKnownInAdvance: boolean
   }
 
-type AddOnchainSendLedgerMetadata = NonIntraledgerLedgerMetadata &
+type AddOnchainSendLedgerMetadata = LedgerMetadata &
   SendAmountsMetadata & {
     hash: OnChainTxHash
     payee_addresses: OnChainAddress[]
     sendAll: boolean
   }
 
+type NonIntraledgerLedgerMetadata = LedgerMetadata & {
+  usd: number // to be removed when "centAmount" takes over
+  fee: Satoshis
+  feeUsd: number // to be removed when "centFee" takes over
+}
```

Comment:
not sure to understand this. aren't we removing this fields?

Reply:
I haven't finished removing them from our Admin transaction types as yet. I actually had to add back some additional (temporary) provisions for this after this comment here https://github.com/GaloyMoney/galoy/pull/2191/commits/7d9ad41413e417dda5a4f2096bcc558dc34873d6